static/js/hashchange.js

```diff
@@ -187,13 +187,37 @@ function do_hashchange(from_reload) {
 // -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- - -- //
 var ignore = {
     flag: false,
-    prev: null
+    prev: null,
+    group: null
 };
 
 function get_main_hash(hash) {
     return hash.replace(/^#/, "").split(/\//)[0];
 }
 
+// different groups require different reloads. The grouped elements don't
+// require a reload or overlay change to run.
+var get_hash_group = (function () {
+    var groups = [
+        ["subscriptions"],
+        ["settings", "administration"]
```

Comment:
I guess technically the second line shoudln't be here since settings isn't merge yet?